libs/output-mapping/src/Storage/TableChangesStore.php

```diff
@@ -5,12 +5,15 @@
 namespace Keboola\OutputMapping\Storage;
 
 use Keboola\OutputMapping\Mapping\MappingFromConfigurationSchemaColumn;
+use Keboola\OutputMapping\Mapping\MappingFromConfigurationSchemaPrimaryKey;
 
 class TableChangesStore
 {
     /** @var MappingFromConfigurationSchemaColumn[] $missingColumns */
     private array $missingColumns = [];
 
+    private ?MappingFromConfigurationSchemaPrimaryKey $primaryKey = null;
```

Comment:
tady bych předal prostě jen MappingFromConfigurationSchemaColumn - takhle mi to přijde překombinovaný a děláš to jen kvůli tomu array_map

Reply:
na callu jsme se domluvili že to necháme